scopes/dependencies/dependency-resolver/dependencies/dependency.ts

```diff
@@ -1,6 +1,8 @@
 import type { DependencySource } from '../policy/variant-policy/variant-policy';
 
-export type DependencyLifecycleType = 'runtime' | 'dev' | 'peer';
+export type WorkspaceDependencyLifecycleType = 'runtime' | 'peer';
+
+export type DependencyLifecycleType = WorkspaceDependencyLifecycleType | 'dev' | 'peer';
```

Comment:
@zkochan you don't need peer here. It's already part of the ws type

Reply:
indeed
updated